app/views/plans/_guidance_choices.html.erb

```diff
@@ -3,23 +3,18 @@
     <li class="checkbox">
       <%= check_box_tag "guidance_group_ids[]", groups[0].id,
                         current_selections.include?(groups[0].id), class: 'guidance-choice' %>
-      <%= form.label org.name, for: groups[0].id, 
-              class: 'inline checkbox-label regular-text guidance-group-label' %>
+      <%= org.name %>
     </li>
   <% elsif groups %>
     <li class="checkbox">
-      <label class="inline checkbox-label regular-text guidance-group-label">
-        <%= org.name %>
-      </label>
-
+      <%= org.name %>
       <ul>
         <% groups.each do |group| %>
           <li class="checkbox">
-            <span class="sublist">└─ </span> 
+            <span class="sublist">└─ </span>
```

Comment:
we can get rid of this symbol if we adopt the proposal from @vyruss raised yesterday.